### DISCUSSION TITLE
Qiita のトップを「トレンド - [全て]タブ」にしてほしい

### DISCUSSION THREAD

**Discussion by uhooi** [increments/qiita-discussions#10] [要望・アイデア・不具合]
私は今まで朝と 17 時にすべてのトレンドをチェックしていました。
いつの間にか「トレンド → 全て」と 2 回もクリックしないと見れないようになり、手間だと感じます。
自分に合った記事がホームに表示されるのも嬉しいのですが、古い記事が多く、私がトップに求めているものとは異なります。

ちなみにトップがホームに変わってから、トレンドをチェックすることがほぼなくなりました。
全てのトレンドをブックマークすればいいいだけの話ですが、つい後回しにしてしまっています 😢

**Comment by daikisuyama** [comment]
自分も@uhooiさんと同じ考えです。
Qiitaのトレンド自体に賛否両論はあると思うのですが、トレンドに好印象を抱いている人は一定数いると思います。
以前のようにトレンドをトップページにするのが難しい場合でも、ホームとトレンドとタイムラインのいずれをトップページにするか設定できるとありがたいです。
また、Qiitaをコミュニティとして盛り上げていくには、ホームよりもトレンドの方が適していると感じます。

**Comment by kskwtnk** [comment]
@uhooi @daikisuyama 

お二人ともありがとうございます！

こちらは要望としてもよくあがっている意見でして、私たちもかなり悩んでいます。
正直な話をするとこのベータテスト期間だけで方針を決めるのは難しいと思っているのですが、本運用まで至った際により多くの人からの意見も聞きながら機能を作っていきたいです。

ふんわりとした返事になってしまい申し訳ないのですが、ホームの在り方については多くのご意見をいただいていて、運営の中でも頻繁に議論になっています。
もしかしたらベータ版として新しいフィードの形式を提供する……なんていうのもあるかもしれません。

少し期間が長くなってしまいそうですが、引き続きご意見いただけたら嬉しいです！よろしくお願いします！

**Reply by uhooi** [reply]
さすがにベータテスト中に方針が決まるとは思っていないので大丈夫ですー！
ただ、なぜおすすめの記事をトップに表示しているか、理由があれば教えていただきたいです。

**Reply by kskwtnk** [reply]
@uhooi 

> なぜおすすめの記事をトップに表示しているか、理由があれば教えていただきたいです。

こちらについてですが、直近だとトレンドタブについては以下のような声が挙がっていました。

- 自分の興味のあることが世の中のトレンドとあまり関連がなく、惹かれる記事がない
- トレンドの性質上、初心者向け・煽り記事・ポエムなどが上がってきやすく、技術の話が少なめ
- 先進的な内容とか突っ込んだ内容の記事が読みたいけど、そういうのはトレンドには上がってこない

Qiitaも10周年を迎えていますので、記事の種別やユーザーさまの属性・レベル感などかなり多様になってきています。
単一のトレンド情報だけを提供していても、様々なユーザーさまにとって「面白い！」と思えるフィードを作れないのではないかと思っていました。
そういうわけで、個人個人の属性にあわせて記事を出した方がより読みたくなるフィードを作れるのではないか？という仮説のもと、ホームフィードをトップに表示しています！
（ただ、更新頻度ですとか、記事の新しさとか……そういった面で当初想定していた世界が作り切れていないのも事実です。）

元々の狙いとしてはこんな感じです！

他にも何か疑問などあれば、お気軽に聞いてください！

**Reply by uhooi** [reply]
当初のねらいを理解できました、ありがとうございます！

**Comment by kyntk** [comment]
他にも意見があるかたがいましたら、コメントいただけますと嬉しいです。
運営からの一方的な意見ではなく、広く意見を集められたらと考えております

**Reply by kokosabu** [reply]
定期的に巡回する系統のものは、RSSリーダーで読めば良いのではないかなと感じます。
( 最近は RSSリーダー は使われなくなってきている?かもしれませんけども )

**Comment by murakami** [comment]
ホームを表示して興味がある記事を探すということをしていませんので、今のところ希望はございません。
ただ、今後グループ機能が用意された際、ホームを表示して調べるということになるかもしれませんね。
その場合、興味があることはグループ単位になっていますので、全てのトレンドが直ぐに表示された方が嬉しいかもしれませんね。
